import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/monitoring/Event.java

```diff
@@ -0,0 +1,42 @@
+package com.google.cloud.healthcare.imaging.dicomadapter.monitoring;
+
+public enum Event implements IMonitoringEvent {
+  STARTED(Constants.prefix + "started"),
+
+  CFIND_REQUEST(Constants.prefix + "cfind_requests"),
```

Comment:
Can we remove the move and cfind errors and add them back in the next PR?

Reply:
Ok